modules/conflict_resolver/jsx/react.conflict_resolver.js

```diff
@@ -0,0 +1,259 @@
+import {Tabs, TabPane} from 'Tabs';
+import FilterForm from 'jsx/FilterForm';
+import formatColumn from './columnFormatter';
```

Comment:
I believe that it was discussed in our reactification workshops/meetings that `formatColumn` should be a function within the index component for the filter/datatable page.

Reply:
@HenriRabalais Please explain more, it currently it works and I'm not understanding the benefit.

Reply:
Simply put, `columnFormatter.js` is a file that contains a function that could be placed in this class instead of in it's own file. It's purely a matter of deprecated organisation which we are trying correct with new reactification PRs.